app/services/CategorisationService.scala

```diff
@@ -39,28 +43,61 @@ class CategorisationService @Inject() (
     recordId: String,
     longerCode: Boolean = false
   )(implicit hc: HeaderCarrier): Future[CategorisationInfo] =
-    ottConnector
-      .getCategorisationInfo(
-        commodityCode,
-        request.eori,
-        request.affinityGroup,
-        Some(recordId),
-        country,
-        LocalDate.now()
-      )
-      .flatMap { response =>
-        CategorisationInfo.build(response, commodityCode, longerCode) match {
-          case Some(categorisationInfo) => Future.successful(categorisationInfo)
-          case _                        =>
-            Future.failed(new RuntimeException("Could not build categorisation info"))
+    profileConnector.getTraderProfile(request.eori).flatMap { profile =>
```

Comment:
Test case could be added for if getTraderProfile fails

Reply:
Added a new test